homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/js/homework.js

```diff
@@ -0,0 +1,58 @@
+function addition(firstValue, secondValue) {
+    return firstValue + secondValue;
+}
+
+function subtraction(firstValue, secondValue) {
+    return firstValue - secondValue;
+}
+
+function multiplication(firstValue, secondValue) {
+    return firstValue * secondValue;
+}
+
+function division(firstValue, secondValue) {
+    return firstValue / secondValue;
+}
+
+function remainder(firstValue, secondValue) {
+    return firstValue % secondValue;
+}
+
+/* function exponentiation(firstValue, secondValue) {
+    return firstValue ** secondValue;
+} */
+
+function calculate(firstValue, secondValue, operation) {
+    let result = 0;
+    const first = Number(firstValue);
+    const second = Number(secondValue);
+    if (first.isNaN || second.isNaN) {
+        alert('Enter a number');
```

Comment:
Something does not work here. This error message should be stored in the `result` and returned immediately in this case 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/9b2cbe17-db9a-416d-b8b6-fe25a3269efe)

Reply:
Same for this case 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/e7e241c6-19f6-449b-9af3-5e42196fe66b)

Reply:
In the beginning, there was "isNaN(first)", and it worked fine, although the linter messaged me with the necessity to change it to "first.isNaN". And yeah, as I see, it doesn't work now. Just decided to inform you about it :)

Reply:
You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
Hey, @Moo0oony, there is one more error case to cover: for empty fields you should show `Enter a number` error
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/532ffd24-4f00-4b78-9ed1-ba108abc0eb3)